packages/cli/src/commonlib/azureLogin.ts

```diff
@@ -456,6 +456,27 @@ export class AzureAccountManager extends login implements AzureAccountProvider {
     }
     throw NotFoundSubscriptionId();
   }
+
+  getAccountInfo(): Record<string, string> | undefined {
+    if (AzureAccountManager.codeFlowInstance.account) {
+      return this.getJsonObject() as unknown as Record<string, string>;
+    } else {
+      return undefined;
+    }
+  }
+
+  getSubscriptionId(): string | undefined {
+    return AzureAccountManager.subscriptionId;
```

Comment:
I suppose the logic in `AzureAccountManager` could be refactored as well. Just leave "Method not implemented." here or add `TODO` to indicate this API is not ready.

Reply:
Add TODO to ```selectSubscription```, other methods work for now.